make/test.mk

```diff
@@ -100,8 +100,8 @@ endif
 
 .PHONY: setup-kubefed
 setup-kubefed:
-	curl -sSL https://raw.githubusercontent.com/codeready-toolchain/toolchain-common/master/scripts/add-cluster.sh | bash -s -- -t member -mn $(MEMBER_NS) -hn $(HOST_NS) -s
-	curl -sSL https://raw.githubusercontent.com/codeready-toolchain/toolchain-common/master/scripts/add-cluster.sh | bash -s -- -t host -mn $(MEMBER_NS) -hn $(HOST_NS) -s
+	curl -sSL https://raw.githubusercontent.com/MatousJobanek/toolchain-common/toolchain-cluster-script/scripts/add-cluster.sh | bash -s -- -t member -mn $(MEMBER_NS) -hn $(HOST_NS) -s
```

Comment:
A reminder :)

Reply:
I need to merge this PR with this change - the reason is that I need to use the version relying on KubeFed for the member operator, but this new version that uses ToolchainCluster for host opererator. So I need to use two versions of the script. I'll replace the URL a bit later - the flow is:
1. merge this PR
2. merge PR changing the script
3. change the URL back to `/codeready-toolchain/` in https://github.com/codeready-toolchain/toolchain-e2e/pull/140
4. merge that e2e member PR